src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -53,6 +62,31 @@ export class AsyncCreatableSelectInput extends React.Component {
     onBlur: PropTypes.func,
     loadOptions: PropTypes.func.isRequired,
     isMulti: PropTypes.bool,
+
+    components: PropTypes.shape({
+      ClearIndicator: PropTypes.func,
+      Control: PropTypes.func,
+      DropdownIndicator: PropTypes.func,
+      Group: PropTypes.func,
+      GroupHeading: PropTypes.func,
+      IndicatorsContainer: PropTypes.func,
+      IndicatorSeparator: PropTypes.func,
+      Input: PropTypes.func,
+      LoadingIndicator: PropTypes.func,
+      Menu: PropTypes.func,
+      MenuList: PropTypes.func,
+      LoadingMessage: PropTypes.func,
+      NoOptionsMessage: PropTypes.func,
+      MultiValue: PropTypes.func,
+      MultiValueContainer: PropTypes.func,
+      MultiValueLabel: PropTypes.func,
+      MultiValueRemove: PropTypes.func,
+      Option: PropTypes.func,
+      Placeholder: PropTypes.func,
+      SelectContainer: PropTypes.func,
+      SingleValue: PropTypes.func,
+      ValueContainer: PropTypes.func,
+    }),
```

Comment:
It would probably also have been fine to just use `PropTypes.object` as this component is only forwarding this prop and not using it itself. `react-select`s prop-types check will warn about invalid props anyways.